src/parser/retail/modules/items/dragonflight/enchants/InvigoratingSporeCloud.tsx

```diff
@@ -0,0 +1,269 @@
+      buff.refreshes += 1;
+    }
+  }
+
+  private onRemoveBuff(event: RemoveBuffEvent) {
+    const buff = this.lastMatchingBuff(event);
+
+    if (buff) {
+      buff.end = event.timestamp;
+
+      this.updateStats(buff.stat, -buff.rank.value, event);
+    }
+  }
+
+  private updateStats(stat: STAT, amount: number, event: ApplyBuffEvent | RemoveBuffEvent) {
+    this.deps.statTracker.forceChangeStats(
+      {
+        [stat]: amount,
+      },
+      event,
+    );
+  }
+
+  private currentHighestSecondaryStat(): STAT {
+    return [
+      {
+        stat: STAT.CRITICAL_STRIKE,
+        value: this.deps.statTracker.currentCritRating,
+      },
+      {
+        stat: STAT.HASTE,
+        value: this.deps.statTracker.currentHasteRating,
+      },
+      {
+        stat: STAT.MASTERY,
+        value: this.deps.statTracker.currentMasteryRating,
+      },
+      {
+        stat: STAT.VERSATILITY,
+        value: this.deps.statTracker.currentVersatilityRating,
+      },
+    ].reduce((acc, stat) => {
+      if (stat.value > acc.value) {
+        return stat;
+      }
+      return acc;
+    }).stat;
+  }
+
+  private lastMatchingBuff(event: ApplyBuffEvent | RefreshBuffEvent | RemoveBuffEvent) {
+    const buff = findLast(this.buffs, (b) => b.source?.id === event.sourceID && b.end == null);
+
+    if (buff == null) {
+      console.error('[InvigoratingSporeCloud] Could not find active buff', event);
+      return;
+    }
+
+    return buff;
+  }
+
+  // ================ STATISTIC ================
+
+  private statisticDropdown() {
+    const rows = this.buffs.map((buff) => {
+      return (
+        <tr key={`${buff.source?.id}-${buff.start}-${buff.end}`}>
+          <td className="text-left">
+            <span className={buff.source ? classColor(buff.source) : undefined}>
+              {buff.source?.name ?? 'Uknown'}
+            </span>
+          </td>
+          <td>
+            <QualityIcon quality={buff.rank.rank} />
+          </td>
+          <td>{buff.rank.value}</td>
+          <td>{getNameTranslated(buff.stat)}</td>
+          <td>
+            {this.owner.formatTimestamp(buff.start ?? this.owner.fight.start_time)} -{' '}
+            {this.owner.formatTimestamp(buff.end ?? this.owner.fight.end_time)}
+          </td>
+        </tr>
+      );
+    });
+
+    return (
+      <table className="table table-condensed">
+        <thead>
+          <tr>
+            <th style={{ textAlign: 'left' }}>Source</th>
+            <th></th>
+            <th>Amount</th>
+            <th>Stat</th>
+            <th>Duration</th>
+          </tr>
+        </thead>
+        <tbody>{rows}</tbody>
+      </table>
+    );
+  }
+
+  private statisticTooltip() {
+    return (
+      <>
+        Friendly healers with the <SpellLink spell={SPELLS.SPORE_TENDER_ENCHANT} /> enchant has a
+        chance to grant you a buff that increases your highest secondary stat. This is is a
+        breakdown of benefit you have gained from this buff.
+      </>
+    );
```

Comment:
i am not sure that this should be a tooltip---its pretty important for contextualizing the number shown.

Reply:
i guess maybe it is okay because there is a separate stat for Spore Tender itself, but a title for the stat like `Personal <SpellLink>Invigorating Spore Cloud</SpellLink> Gain`

Reply:
Just two tiny grammar notes

- has -> have 
- breakdown of benefit -> breakdown of the benefit

Reply:
> Just two tiny grammar notes
> 
> * has -> have
> * breakdown of benefit -> breakdown of the benefit

Oh thanks a lot! Fixed!

> i guess maybe it is okay because there is a separate stat for Spore Tender itself, but a title for the stat like `Personal <SpellLink>Invigorating Spore Cloud</SpellLink> Gain`

![image](https://github.com/WoWAnalyzer/WoWAnalyzer/assets/13413409/98e2c248-15ba-45c2-bd71-b251b961ca05)

Linebreak becomes a bit of a nightmare.

What do you think of this? 

![image](https://github.com/WoWAnalyzer/WoWAnalyzer/assets/13413409/6fd065d3-dce3-4765-b34c-96bc3427e339)